madmom/io/audio.py

```diff
@@ -96,6 +97,10 @@ def _ffmpeg_call(infile, output, fmt='f32le', sample_rate=None, num_channels=1,
         Maximum length in seconds to decode.
     cmd : {'ffmpeg','avconv'}, optional
         Decoding command (defaults to ffmpeg, alternatively supports avconv).
+    rg_mode : {'track','album', None}, optional
```

Comment:
Default parameters should be listed first.
```suggestion
    replaygain_mode : {'None', 'track','album'}, optional
```

Reply:
Oh OK didn't realise that; I haven't used this documentation style really - more used to reST style (or increasingly just Python 3.6+ type annotations a simple docstring). I'll update it - thanks